configure.ac

```diff
@@ -165,6 +165,14 @@ AC_ARG_WITH([ecmult-window], [AS_HELP_STRING([--with-ecmult-window=SIZE|auto],
 )],
 [req_ecmult_window=$withval], [req_ecmult_window=auto])
 
+AC_ARG_WITH([ecmult-gen-precision], [AS_HELP_STRING([--with-ecmult-gen-precision=2|4|8|auto],
+[Precision bits to tune the precomputed table size for signing.]
+[The size of the table is 32kB for 2 bits, 64kB for 4 bits, 512kB for 8 bits of precision.]
+[A smaller table size usually results in slower signing.]
```

Comment:
> With `./configure --disable-openssl-tests --with-ecmult-gen-precision=8` (but not `2` and `4`) and `valgrind ./exhaustive_tests` I get millions of warnings like

There is no issue if you call it like this:

```valgrind --max-stackframe=2097912 ./exhaustive_tests```

[Reference](http://valgrind.org/docs/manual/manual-core.html):

> -max-stackframe=<number> [default: 2000000]
>    The maximum size of a stack frame. If the stack pointer moves by more than this amount then Valgrind will assume that the program is switching to a different stack.
>    You may need to use this option if your program has large stack-allocated arrays. 

Seems like the big option just needs more stack space. It's a valgrind heuristic only and the tests run fine otherwise.

Reply:
Oh indeed, thanks!